changelog.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## What's New (09/12/2017)
+
+### Resolved issues/Bug fixes
+-	Better camelcased suggestions
+-	Better handling of empty operationIds in OpenAPI specs
+-	Case insensitive checks for top level resource properties
+-	Better messaging for tracked resources related rules
+
+### New validation rules
+-	ArraySchemaMustHaveItems – A schema of array type must always contain an items property. without it, AutoRest will fail to generate an SDK. Documentation [link](https://github.com/Azure/azure-rest-api-specs/blob/8ac36944d0ab7f6d07daf077a99bdae873d5e3f6/documentation/openapi-authoring-automated-guidelines.md#L88)
```

Comment:
link doesn't seem to point to the rule (but top of doc) if I copy/paste it, should we link to the anchor https://github.com/Azure/azure-rest-api-specs/blob/8ac36944d0ab7f6d07daf077a99bdae873d5e3f6/documentation/openapi-authoring-automated-guidelines.md#r2009-arrayschemamusthaveitems ?